src/app/Http/Controllers/DeleteController.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Support\Facades\DB;
+
+class DeleteController extends Controller
+{
+    public function __invoke()
+    {
+        DB::table('tasks')->where('id', '=', $_GET['task'])->delete();
```

Comment:
It is not a good idea to take the parameter using $_GET in laravel.
Add as parameter to invoke `__invoke(Request $request)` and retrieve the parameter from there `$request->input('task')`